examples/react-native/src/features/Authenticator/Demo/Example.tsx

```diff
@@ -1,19 +1,17 @@
 import React from 'react';
-import { StyleSheet, View } from 'react-native';
+import { Button, StyleSheet, View } from 'react-native';
 
 import { Authenticator, useAuthenticator } from '@aws-amplify/ui-react-native';
 import { Amplify } from 'aws-amplify';
 
-import { Button } from '../../../ui';
-
 // replace with actual amplify config from environments
 // import config from '../../../aws-exports';
 
 Amplify.configure({});
 
 function SignOutButton() {
   const { signOut } = useAuthenticator();
```

Comment:
Should we use 
```suggestion
  const { signOut } = useAuthenticator(({ signOut }) => [signOut]);
```
for performance reasons?

Reply:
Since it's an basic example i don't think we need to go in optimization

Reply:
Understood, resolving.